apis/clusters/v1beta1/postgresql_types.go

```diff
@@ -88,7 +88,13 @@ type PgSpec struct {
 
 // PgStatus defines the observed state of PostgreSQL
 type PgStatus struct {
-	ClusterStatus `json:",inline"`
+	ClusterStatus  `json:",inline"`
+	DefaultUserRef `json:",inline"`
+}
+
+type DefaultUserRef struct {
```

Comment:
Postgresql users work a bit different compare to other users, you can see this in old code, so did you check the cluster connection after removing the old code? 
Did everything work as before?

Reply:
Everithing works properly